torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -130,7 +148,7 @@ class ModuleCloneHelper {
         r.register_attribute(
             type->getAttributeName(i),
             type->getAttribute(i),
-            inplace ? s : s.deepcopy(memo),
+            inplace || s.isCapsule() ? s : s.deepcopy(memo),
```

Comment:
Should this change be part of a separate PR?

Reply:
this change is not needed, removed now